test/api/v3/integration/tasks/DELETE-tasks_id.test.js

```diff
@@ -16,7 +16,7 @@ describe('DELETE /tasks/:id', () => {
     let task;
 
     beforeEach(() => {
-      return user.post('/tasks', {
+      return user.post('/tasks?tasksOwner=user', {
```

Comment:
As we had discussed earlier my idea was to have /tasks/user and /tasks/challenge/userId. I had to use a query parameter because otherwise the route handler would intercept requests going to /tasks/taskId. I think the parameter name could be changed, I don't like it either. But regarding the user being default I don't understand why it should be privileged over challenges (and in future other types). In my opinion it also makes the API behavior clearer